server-kotlin/resources/application.conf

```diff
@@ -12,7 +12,9 @@ ktor {
                jdbcUrl = ${?JDBC_URL}
                driver = org.postgresql.Driver
                dbUser = "postgres"
+               dbUser = ${?DB_USER}
```

Comment:
isn't this duplicated?

Reply:
or the latter value overrides the previous one in case?

Reply:
Exactly, https://ktor.io/servers/configuration/environments.html#applicationconf Ktor FTW